### PR TITLE
fixed member positions

### PR DESCRIPTION
**Pull request by hector0513**
image

### CODE REVIEW

File: src/drawing/drawGoalCard.js [hunk 1]
```diff
@@ -158,9 +158,11 @@ export default function render(
     // assume that it will be drawn the next time 'render' is called
     // if it isn't already set
     if (!img) return
+
     // adjust the x position according to the index of this member
     // since there can be many
-    const xImgDraw = x + goalWidth - (index + 1 * avatarWidth + avatarSpace)
+    const xImgDraw =
+      x + goalWidth - (index + 1) * (avatarWidth + avatarSpace / 2)
```
Comment (Connoropolous) [inline src/drawing/drawGoalCard.js]: can you go find `avatarSpace` in the code, and make it a few pixels less?

**Review by Connoropolous — CHANGES_REQUESTED** [review]
Great! I guess it was just a bug

File: src/drawing/drawGoalCard.js (hunk 1 above)
Comment (Connoropolous) [reply src/drawing/drawGoalCard.js]: and remove the `/ 2`

**Comment by hector0513** [comment]
image

File: src/drawing/drawGoalCard.js (hunk 1 above)
Comment (hector0513) [reply src/drawing/drawGoalCard.js]: ok

**Review by Connoropolous — APPROVED** [review]

**Pull request merged**